src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -1067,6 +1067,48 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
             JasmineGraphServer *jasmineServer = new JasmineGraphServer();
             jasmineServer->inDegreeDistribution(graphID);
 
+            int result_wr_done = write(connFd, DONE.c_str(), FRONTEND_COMMAND_LENGTH);
+            if (result_wr_done < 0) {
+                frontend_logger.log("Error writing to socket", "error");
+                loop = true;
+                continue;
+            }
+            result_wr_done = write(connFd, "\r\n", 2);
+            if (result_wr_done < 0) {
+                frontend_logger.log("Error writing to socket", "error");
+                loop = true;
+                continue;
+            }
+        } else if (line.compare(OUT_DEGREE) == 0) {
```

Comment:
Why do we add the out degree distribution calculation logic in this PR? This PR is supposed to be only on attribute storage.